workflows/nowsecure.yml

```diff
@@ -1,72 +1,55 @@
-# An example workflow using the NowSecure action that builds an Android application
-# in the "build" phase and then runs it against NowSecure in the "scan" phase.
-# This action is run each time a commit is pushed to the "main" branch.
+# Copyright © 2022 NowSecure Inc.
+#
+# SPDX-License-Identifier: MIT
 
-name: "NowSecure"
+name: NowSecure
 
 on:
   push:
-    branches: [ main ]
-  pull_request:
-    branches: [ main ]
-  workflow_dispatch:
-    branches: [ main ]
+    branches:
+      - "*"
 
 jobs:
-  # EXAMPLE: Build and upload an Android application to GH artifacts.
-  build:
+  scan:
     runs-on: ubuntu-latest
+    outputs:
+      report_id: ${{ steps.uploadAndScan.outputs.report_id }}
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
 
-      - name: Install Java
-        uses: actions/setup-java@v2
-        with:
-          java-version: "11"
-          distribution: "adopt"
-          cache: "gradle"
-
-      - name: Validate Gradle wrapper
-        uses: gradle/wrapper-validation-action@v1
+      - name: Install ripgrep
+        run: sudo apt-get install -y ripgrep
 
-      - name: Build application
-        run: ./gradlew assembleDebug --stacktrace --no-daemon
+      - name: Get BitBar APK
```

Comment:
if I understand correctly this is the example action we are recommending customers start from as a template. Therefore I think we need to reconsider the change here as most customers will likely not be working from bitbar's APK rather one they are building themselves. Did we intend to make these changes? 
https://github.com/nowsecure/nowsecure-action/pull/9/files#diff-8d7ab5a2152046673e8019a41206b34e3db379471c9e0d74c236cda3a4bfeef7L23-L31